libs/contentful-extensions/translation/src/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+  </head>
+  <body>
+    <div id="root"></div>
+    <script type="text/javascript" src="./index.tsx"></script>
+  </body>
+</html>
```

Comment:
This file is exactly as it was in the old `contentful-translation-extension/src`. Not sure why github is not picking up on it being simply moved